recipes/glibmm/all/conanfile.py

```diff
@@ -75,11 +75,11 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("glib/2.77.2")
+        self.requires("glib/2.78.0", transitive_headers=True, transitive_libs=True)
```

Comment:
@AndreyMlashkin Do we need to add `transitive_libs=True`? I saw that the CI complained about the missing binaries but not any linking errors. Could we remove it and see the result? Otherwise, we can revert it for sure.

```suggestion
        self.requires("glib/2.78.0", transitive_headers=True)
```